hoot-core/src/main/cpp/hoot/core/cmd/PaintNodesCmd.cpp

```diff
@@ -242,7 +242,7 @@ class PaintNodesCmd : public BaseCommand
         const int32_t* row = mat.ptr<int32_t>(y);
         for (int x = 0; x < qImage.width(); x++)
         {
-          double v = log(row[x] + 1) / log(maxValue);
+          double v = log1p(row[x]) / log(maxValue);
```

Comment:
Come to find out `log1p(x)` is more accurate than `log(x + 1)`.  This made minor changes in some of the unit tests for the better (minutely but better).